osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/GetOutagesCommandExecutor.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2020 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
```

Comment:
This license header is sometimes added with, sometimes without this "empty line" (just the asterisk). It looks a bit sloppy, and I think leaving the empty line out best fits the guidelines.